frontend/src/components/MXEndpoint.js

```diff
@@ -55,14 +88,46 @@ function MXEndpoint({
                 {title}
               </Text>
             </div>
-            <div className="mt-8">
+            {showNotice && (
+              <Text as="ParagraphSmall" color="secondary" tag="p">
+                Notice: The first two parts were completed automatically in the widget because it was configured to {getNotice()}
+              </Text>
```

Comment:
As mentioned in the other comment, you may be able to add the link here
```suggestion
              <Text as="ParagraphSmall" color="secondary" tag="p">
                Notice: The first two parts were completed automatically in the widget because it was configured to
                <span>
                  <a href="https://docs.mx.com/api#connect_request_a_url" target="_blank" rel="noreferrer">{getNoticeMessage()}</a>
                </span>
              </Text>
```